nltk/text.py

```diff
@@ -30,6 +30,7 @@
 from nltk.collocations import BigramCollocationFinder
 from nltk.compat import python_2_unicode_compatible
 
+ConcordanceLine = namedtuple('ConcordanceLine', 'left query right offset left_print right_print line')
```

Comment:
Please make this a list:
```python
['left', 'query', 'right', 'offset', 'left_print', 'right_print line']
```

Reply:
I think list is not a good idea here. Following the "explicit is better than implicit" rule,  namedtuple are more appropriate, because ConcordanceLine is sort of an `enum` class equivalence as in C or Java. 

It also provides a more readable interface when users calls `ConcordanceLine.left` instead of `concordance_line[0]`.

Reply:
Sorry I wasn't clear here, I meant instantiate the namedtuple using a list:

```Python
namedtuple('ConcordanceLine', ['left', 'query', 'right', 'offset', 'left_print', 'right_print line'])
```
Because I think the spaces make it harder to distinguish the individual elements of the named tuple.

Reply:
Ah yes, a list would be easier to read and maintain the code. Pardon the laziness to type the attributes in strings =)

I've made the changes accordingly.